### ISSUE TITLE
Improve Xcom-LAN IP address discovery

### ISSUE THREAD

**Issue by ankohanse** [ankohanse/hass-studer-xcom#4]
As described in the readme, the Xcom-LAN / Moxa needs to be configured to connect to the HA-integration.
This requires discovering the IP-address of the Xcom-LAN via a Moxa discovery tool. However, downloading and using that tool might scare off users that are interested in the integration.

I suspect that that the discovery tool only does a scan of all local IP addresses (192.168.x.1 to 192.168.x.255), tries to connect to each IP address via a specific port and check the response. Possibly it only just tries to open the configuration page of the Xcom-LAN (http request to port 80) and tries to detect a certain string or header in the response.

It would be nice if the setup of the configuration can mimick this discovery so that it can directly open the Xcom-LAN configuration page for the user.

**Comment by ankohanse** [comment]
Moxa Device Search Utility turns out to do a UDP broadcast to some specific ports. See attachment for details of network capture.
[moxa discovery.txt](https://github.com/user-attachments/files/17007919/moxa.discovery.txt)


**Alternatives:**

- TCP connect to port 80 (config website, can be disabled)

   Look for response header 'Server: MoxaHTTP/2.2'
   or <HTML><HEAD><TITLE>Nport Web Console</TITLE></HEAD/></HTML>

  note: HTTPS (port 443) seems not supported


- telnet to 192.168.88.x	(can be disabled)
  check if response contains 'NPort'

```
-----------------------------------------------------------------------------
Model name       : NPort 5110A
MAC address      : 00:90:E8:3C:F8:7E
Serial No.       : 2002
Firmware version : 1.1.31 Build 13111911
System uptime    : 21 days, 15h:58m:07s
-----------------------------------------------------------------------------
<< Main menu >>
  (1) Basic settings
  (2) Network settings
  ...
```

- SNMP	(can be disabled)


Since the goal is to open the Web Config page for the user, a search via TCP port 80 seems most usefull

**Comment by ankohanse** [comment]
This is now implemented and part of release v2024.09.5

Within the setup wizard, after the port is chosen, the wizard will continue to try and connect to Xcom and discover available Studer devices. If Xcom connect fails during this step, it will try to find the url for the Moxa Web Config and present that on the page for the port selection.
